lib/brexit_checker/actions.yaml

```diff
@@ -424,12 +461,14 @@ actions:
   guidance_link_text: Changes to copyright law from 1 January 2021
   guidance_url: https://www.gov.uk/guidance/changes-to-copyright-law-after-the-transition-period
   criteria:
-  - any_of:
-    - media
-    - digital
-    - telecoms
-    - personal-eu-org-provide
-    - ip-copyright
+  - all_of:
+    - any_of:
+      - media
+      - digital
+      - telecoms
+    - any_of:
+      - personal-eu-org-provide
+      - ip-copyright
```

Comment:
this is not on the trello card, so we should probably not commit this update here.

Reply:
Removed